### PR TITLE
Split the Status Node guideline

### PR DESCRIPTION
**Pull request by jinhojang6**
Split the Status Node guideline.

Tested at 
- https://dev-lang.status.im/technical/status_node_step_by_step.html
- https://dev-lang.status.im/technical/others/deploy_ec2_instance.html

### CODE REVIEW

**Review by jakubgs — CHANGES_REQUESTED** [review]
Maybe we can add an `Other` or `Misc` sidebar section where the link to AWS EC2 setup guide can be added.

I'd also add a link to the quickstart guide at the beginning and end of the AWS EC2 guide.

**Comment by jinhojang6** [comment]
Updated the PR

File: source/technical/others/deploy_ec2_instance.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+---
+id: deploy_ec2_instance
+title: Deploy an AWS EC2 instance
+description: Set up an Ubuntu machine to run a Status Node
+---
+
+## Deploy an AWS EC2 instance
+
+This guide will walk you through how to set up an Ubuntu machine to run a Status Node. Afterwards you can follow this [quickstart guide](../status_node_step_by_step.html) to configure the Status Node.
```
Comment (jakubgs) [inline source/technical/others/deploy_ec2_instance.md:9]: This uses the "quickstart" word. Also, the folder is still called `status_node_quick_start`.

File: source/technical/status_node_step_by_step.md [hunk 2]
```diff
@@ -200,7 +94,7 @@ will return IP addresses
 
 <br/>
 
-### 20. Check Waku envelopes
+### 9. Check Waku envelopes
 
 - `curl -s localhost:9090/metrics | grep '^waku_envelopes_received_total'`
```
Comment (jakubgs) [inline source/technical/status_node_step_by_step.md:100]: I'd add another final line here referencing the Step-by-Step guide again.

**Review by jakubgs — CHANGES_REQUESTED** [review]
Almost done, the folder for images has old name.

**Comment by jinhojang6** [comment]
@jakubgs Updated the PR excluding the comment `I'd add another final line here referencing the Step-by-Step guide again.`

**Comment by jinhojang6** [comment]
Added `If you successfully deployed an EC2 instance, please follow this [Status Node Step-by-Step guide](../status_node_step_by_step.html) to configure a Status Node.` on `deploy_ec2_instance.md`

**Review by jakubgs — APPROVED** [review]

**Pull request merged**